config/locales/en.yml

```diff
@@ -327,7 +327,7 @@ en:
       button: 'Preview job listing'
       page_title: 'Preview job listing for %{school}'
       summary:
-        heading: 'Preview of %{title}'
+        heading_html: 'Preview of &#8216;%{title}&#8217; job listing'
```

Comment:
`heading: Preview of '%{title}' job listing` if you prefer. We don't use fancy apostrophes anywhere else on the site yet as far as I know

Reply:
I thought the straight apostrophes looked ugly.

Reply:
<img width="773" alt="Screenshot 2020-06-15 at 18 19 59" src="https://user-images.githubusercontent.com/60350599/84687094-0c237800-af35-11ea-9c3f-abf94278f729.png">

For your consideration

Reply:
Tomayto tomahto, I'm happy either way if UX/Content are happy